packages/core/src/components/way-input/way-input.scss

```diff
@@ -0,0 +1,92 @@
+:host {
+  /**
+    * @prop --border-radius: Border radius of the input
+    * @prop --border-width: Border width of the button
+    * @prop --border-style: Border style of the button
+    * @prop --border-color: Border color of the button
+    */
+  display: inline-block;
+
+  width: auto;
+
+  font-family: var(--way-font-family, inherit);
+  font-size: var(--way-font-size);
+  line-height: 1.25rem;
+
+  user-select: none;
+  vertical-align: top;
+  vertical-align: -webkit-baseline-middle;
+  pointer-events: auto;
+
+  font-kerning: none;
+
+  --border-radius: 5px;
+  --border-width: 3px;
+  --border-color: #4598d8;
+  --border-style: solid;
+}
+
+.input {
+  display: block;
+  appearance: none;
+  background-color: var(--way-form-background-color);
```

Comment:
**suggestion**: Wat mij betreft is dit gewoon `var(--way-color-white);` en hoeft hier niet een specifieke token voor te komen (maar het zou kunnen, maar dan dus graag `--way-form-element-background-color` en dan moet ik die later ook even gaan toepassen in oa de way-select.